experimental/GITFans/GITFans.jl

```diff
@@ -0,0 +1,601 @@
+
+    return (hash_list, edges)
+end;
+
+
+orbits_of_maximal_GIT_cones(orbit_list, hash_list, matrix_action) =
+    orbit_cone_orbits( map(x -> hash_to_cone(orbit_list, x), hash_list), matrix_action);
+
+
+function hashes_to_polyhedral_fan(orbit_list, hash_list, hom)
+    # translate the descriptions of the orbit repres. of maximal cones
+    # to cone objects
+    result_cones = map(x -> hash_to_cone(orbit_list, x), hash_list)
+
+    # expand their orbits
+    expanded = orbit_cone_orbits(result_cones, hom)
+    maxcones = vcat(expanded...)
+
+    # the defining rays for all maximal cones
+    rays_maxcones = [[convert(Vector{Rational{BigInt}}, cone.RAYS[i, :])
+                      for i in 1:size(cone.RAYS, 1)]
+                      for cone in maxcones]
+
+    # the defining rays for the orbit representatives of maximal cones
+    rays_result_cones = [[convert(Vector{Rational{BigInt}}, cone.RAYS[i, :])
+                      for i in 1:size(cone.RAYS, 1)]
+                      for cone in result_cones]
+
+    # the set of rays
+    allrays = sort(collect(Set(vcat(rays_maxcones...))))
+
+    # the indices of rays that belong to each maximal cone (0-based)
+    index_maxcones = [sort([findfirst(x -> x == v, allrays)-1
+                            for v in rays])
+                      for rays in rays_maxcones]
+
+    # the indices of rays that belong to each repres. cone (0-based)
+    index_result_cones = [sort([findfirst(x -> x == v, allrays)-1
+                            for v in rays])
+                      for rays in rays_result_cones]
+
+    # The fan object without information about the symmetry group
+    # could now be constructed as follows.
+    # # return Polymake.fan.PolyhedralFan(INPUT_RAYS = hcat(allrays...)',
+    # #                                   INPUT_CONES = index_maxcones)
+
+    # In order to create a fan object with symmetry information, prepare
+    # - the matrix of rays,
+    # - the permutation generators of the symmetry group G acting on the rays,
+    #   computed from the matrix action,
+    # - representatives of the G-orbits on the maximal cones,
+    #   each given by its defining rays, via the row indices (zero based)
+    #   in the matrix of rays.
+    pm_rays = Polymake.Matrix(hcat(allrays...)')
+    matgens = [image(hom, x) for x in gens(domain(hom))]
+    mats_transp = [GAP.gap_to_julia(Matrix{Rational{BigInt}}, x.X)' for x in matgens]
+
+    # Note that the matrices have been transposed because we use
+    # matrix times vector multiplication.
+    actfun = function(ray, mat)
+      ray = mat * ray
+      # normalize the image vector
+      for i in 1:length(ray)
+        if ray[i] != 0
+          return ray // ray[i]
+        end
+      end
+      error("a zero vector should not appear")
+    end
+
+    rewr = [as_permutation(x, allrays, actfun, ==) for x in mats_transp]
+
+    rewr2 = [Vector(x) .- 1 for x in rewr]  # zero based
+    pm_gens = @Polymake.convert_to Array{Array{Int}} rewr2
```

Comment:
Julia best practice is to write `MODULE.@MACRO` not `@MODULE.MACRO`
```suggestion
    pm_gens = Polymake.@convert_to Array{Array{Int}} rewr2
```